codegen/winterfell/Cargo.toml

```diff
@@ -1,7 +1,15 @@
 [package]
 name = "air-codegen-winter"
 version = "0.1.0"
+description="Winterfell code genreator for the AirScript languge"
```

Comment:
nit: There's are a couple of typo. It should be "generator" instead of "genreator" and "language" instead of "languge"

Reply:
Sorry for all the typos! (apparently, my spell-checker doesn't work in `.toml` files). Fixed!